test/Microsoft.ML.TestFramework/DataPipe/TestDataPipe.cs

```diff
@@ -1475,6 +1477,10 @@ public void TestLDATransform()
                 Assert.True(resultThirdRow.GetItemOrDefault(1) == 0);
                 Assert.True(resultThirdRow.GetItemOrDefault(2) == 1.0);
             }
+            finally
+            {
+                ldaTransformer.Dispose();
```

Comment:
Why not use a `using`?

Reply:
The type does not implement `IDisposable`.

Reply:
Should it?

Reply:
It could, but that would be part of #4621 if the change is made.